dist/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>RecipeNation</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+<script defer src="main.js"></script></head>
+<body>
+    <header>
+        <nav class ='nav-link'>
+            <ul class ="d-flex">
+                <li><a href="#"><img src="assets/main-logo2a04e92b103dd4a80351.png" alt="logo"></a></li>
+                <li><a href="#">Meals</a></li>
```

Comment:
- [ ] As you did with comments, you should have a counter for the number of meals on the page, kindly fix this issue. 😄 
    - Remember that even if the API gives you the number of items, you should create a specific function for that purpose.
    - And don't forget that you should test this function. 🚀 
    ![image](https://user-images.githubusercontent.com/60789317/137360083-b90e4c6c-ac2b-4953-9aa5-ef798750c72e.png)

Reply:
Well, if that is the case, please add that number to the navbar, like is showed in the image, and don't worry about the counter function, just make it similar to the template. 😄

Reply:
@helman101 Thank you very much for your kind review bro I will do it this way